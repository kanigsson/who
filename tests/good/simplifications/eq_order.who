type unit
logic = ['a||] : 'a -> 'a -> prop
logic -> : prop -> prop -> prop
logic < : int -> int -> prop
axiom z :
  forall (x : int).
    x < 0 -> x = 0 -> 0 = x
