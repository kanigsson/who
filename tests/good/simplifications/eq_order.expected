logic = ['a||] : a -> a -> prop
logic ->  : prop -> prop -> prop
logic <  : int -> int -> prop
axiom z : True
